apps/mobile/src/navigation/TabsNavigator.js

```diff
@@ -56,7 +56,7 @@ export default function TabsNavigator () {
     <Tabs.Navigator {...navigatorProps}>
       <Tabs.Screen name='Home Tab' component={HomeNavigator} />
       <Tabs.Screen name='Search Tab' component={SearchNavigator} />
-      <Tabs.Screen name='Messages Tab' component={MessagesNavigator} />
+      <Tabs.Screen name='Messages Tab' component={MessagesNavigator} options={{ tabBarBadge: 999 }} />
```

Comment:
Whats this do?

Reply:
A bookmark for badges to come. Quite unmissable in the UI